src/main/resources/static/openapi/EducationAndWorkPlanAPI.yml

```diff
@@ -19,8 +19,25 @@ paths:
   #
 
   #
-  # Goals
+  # Action Plans
   # --------------------------------------------------------------------------------
+  '/action-plans/':
+    post:
+      summary: Retrieves a list of Action Plans for a given list of Prisoners.
+      description: Retrieves a list of Action Plans for a given list of Prisoners.
+      tags:
+        - Action Plan
+      responses:
+        '201':
+          $ref: '#/components/responses/ActionPlanSummaryList'
+        '400':
+          $ref: '#/components/responses/400ErrorResponse'
+        '404':
```

Comment:
Could this return a 404? Are you proposing that the impl has some logic that says if none of the supplied prison numbers have an action plan then its a 404? (personally I'd say that was still a 200 with an empty array response?)

Reply:
Good point, I'll remove it 👍